src/views/Player.vue

```diff
@@ -226,9 +237,22 @@ export default {
           this.isFullscreen = false;
           this.sessionId = res.data.sessionId;
           this.browser = res.data.userAgent["browser"]["family"];
-          this.configs = res.data.configData;
-          this.isTutorialComplete = this.configs.tutorial.isComplete;
-          this.tutorialProgress = this.configs.tutorial.progress;
+          this.userConfigs = res.data.userConfig;
+          this.isTutorialComplete = this.userConfigs.tutorial.isComplete;
+          this.tutorialProgress = this.userConfigs.tutorial.progress;
+
+          // fetching plio config and verifying it with the component-properties.json
+          if('player' in res.data.plioConfig) {
+            this.plioPlayerConfig = res.data.plioConfig['player']
+          }
+          else this.plioPlayerConfig = {}
+
+          for (const [feature, details] of Object.entries(this.componentProperties)) {
+              this.plioPlayerConfig[feature] = details
+          }
+
+          this.progressBarInfo['config'] = this.plioPlayerConfig['progress_bar']
+          this.progressBarInfo['completionPercent'] = 0
```

Comment:
1. Storing the player specific config data into a variable
2. Iterating through the system wide config, and if anything is found, replace the info of that key into the `plioPlayerConfig`
3. set the `progress_bar` info to another object named `progressBarInfo` which will be passed on to the `PlioQuestion.vue`
4. Set the `completionPercent` as 0 as a default